compiler/rustc_mir/src/interpret/operand.rs

```diff
@@ -79,6 +79,16 @@ impl<'tcx, Tag: Provenance> Immediate<Tag> {
     pub fn to_scalar(self) -> InterpResult<'tcx, Scalar<Tag>> {
         self.to_scalar_or_uninit().check_init()
     }
+
+    #[inline]
+    pub fn to_scalar_pair(self) -> InterpResult<'tcx, (Scalar<Tag>, Scalar<Tag>)> {
+        match self {
+            Immediate::ScalarPair(val1, val2) => Ok((val1.check_init()?, val2.check_init()?)),
+            Immediate::Scalar(..) => {
+                bug!("Got a scalar where a wide pointer was expected")
```

Comment:
```suggestion
                bug!("Got a scalar where a scalar pair was expected")
```
Wide pointers are not the only scalar pairs.

Reply:
Done. Thanks!